user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -574,7 +574,7 @@ def get_costs_for_single_app(df_row, estimator: SavingsEstimator) -> pd.Series:
                                                                             df_row['Estimated GPU Duration'])
             cpu_cost = (100 - self.ctxt.get_ctxt('cpu_discount')) / 100 * raw_cpu_cost
             gpu_cost = (100 - self.ctxt.get_ctxt('gpu_discount')) / 100 * raw_gpu_cost
-            est_savings = 100.0 - ((100.0 * gpu_cost) / cpu_cost)
+            est_savings = 100.0 - ((100.0 * gpu_cost) / cpu_cost) if cpu_cost > 0 else 0
```

Comment:
Should we additionally add a note/comment in the output saying that CPU cost calculation was 0 as that likely indicates an issue with provided cluster shape from the user?

Reply:
Yes, that makes sense. Let me add that.